docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst

```diff
@@ -15,11 +15,13 @@ Whenever you want to see a detailed explanation of any command, simply add the `
 For more detailed information about ``verdi``, you can refer to the `online documentation <https://aiida-core.readthedocs.io/en/latest/verdi/verdi_user_guide.html>`_.
 
 
+.. _setup_verdi_quicksetup:
+
 Setting up a profile
 --------------------
 
 After you install AiiDA, the next important step is to create a "profile".
-Each AiiDA installation supports multiple profiles, each of which can have its very own database, that won't interact with the other.
+Each AiiDA installation supports multiple profiles, each of which can have its very own database.
 In this way, one can view an AiiDA profile as a representation of an individual project you want to work on.
 
 The official way of setting up a new profile is through the command ``verdi setup``.
```

Comment:
Can we just use `verdi quicksetup` here and put the information about `verdi setup` in a `.. note::` that also links to the corresponding section of the documentation (setting up the database)?